File: 3617-FindTheOriginalTypedStringI/3617-FindTheOriginalTypedStringI.c
// Last updated: 6/22/2026, 12:48:47 PM
int possibleStringCount(char* word) {
    int count = 1;
    int index = 1;
    while(word[index] != '\0'){
        if(word[index] == word[index-1]) count++;
        index++;
    }

    return count;
}

File: 3617-FindTheOriginalTypedStringI/3617-FindTheOriginalTypedStringI.py
# Last updated: 6/22/2026, 12:48:42 PM
class Solution:
    def possibleStringCount(self, word: str) -> int:
        count = 1
        length = len(word)
        for i in range(1, length):
            if word[i-1] == word[i] :
                count += 1
        return count